test-scripts/integrationTest.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Please bump the year to 2019 if you have a chance

Reply:
sure and updated the tslint